workers/unity/Packages/com.improbable.gdk.core/Systems/CustomSpatialOSSendSystem.cs

```diff
@@ -21,19 +21,19 @@ public abstract class CustomSpatialOSSendSystem<T> : ComponentSystem where T : I
 
         protected WorkerSystem WorkerSystem;
 
-        private SpatialOSSendSystem spatialOSSendSystem;
+        private ReactiveComponentSendSystem reactiveComponentSendSystem;
```

Comment:
do we need the reactive component send system for custom send systems?
Why not use the ComponentSendSystem?

Reply:
Refactor error. Will fix.